README.md

```diff
@@ -1 +1,138 @@
-# Budget-app-manager
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>Budget-app</b></h3>
+</div>
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Budget-app](#-budget-app)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+  - [Live demo](#live-demo)
+  - [Presentation](#presentation)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 Budget-app<a name="about-project"></a>
+
+This budget project is about a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- RUBY
+- RUBY ON RAILS
+- JAVASCRIPT 
+- HTML 
+- CSS
+
+### Key Features <a name="key-features"></a>
+
+- [ ] **Users can create a account.**
+- [ ] **Users can add a category of tem**
+- [ ] **Users can add transactions to their categories**
+- [ ] **Users can see what they spent on a transaction**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get started with this app, you will need to have Ruby and Rails installed on your computer. Once you have Ruby and Rails installed.
+
+## Live demo
+   [live demo](https://budget-app-jncr.onrender.com)
+
+
```

Comment:
- [ ] I couldn't find the presentation video link in the README file or PR description. Please make sure to record a video presenting the features of the project you built as requested in the project requirements. It is a video with a maximum length of 5 minutes. The content of the video should include:
![image](https://github.com/tchasinga/Budget-app-manager/assets/83803414/7d8f6ba8-1e45-4adf-8989-ae8562d8cdd2)